feature/gribi/otg_tests/weighted_balancing_test/setup_test.go

```diff
@@ -206,12 +206,6 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 			t.Fatalf("No address found for port %v", dp)
 		}
 	}
-
```

Comment:
Line 210-214 are expected to delete?

Reply:
Yes. The deviation changes are removed, as `AssignToNetworkInstance` require reference to the `testing.T` instead of direct interface `testing.TB`.  So we plan to move these deviation changes for following 4 files as new PR, where we would require `testing.TB` interface change as well - 

> 1. feature/experimental/isis/ate_tests/internal/session/session.go
> 2. feature/experimental/isis/otg_tests/internal/session/session.go
> 3. feature/gribi/ate_tests/weighted_balancing_test/setup_test.go
> 4. feature/gribi/otg_tests/weighted_balancing_test/setup_test.go

Reply:
Sounds good.